packages/components/app/styles/components/app-footer.scss

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// app-footer
+//
+
+// WIP: names, etc. may change
+:root {
+  --token-app-footer-text-color: var(--token-color-foreground-faint);
+  --token-app-footer-border-color: var(--token-color-border-primary);
+}
+
+// NOTE: Styles are for dev purposes currently, visual designs are NOT final
+.hds-app-footer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  justify-content: flex-end;
+
+  padding: 24px;
+  color: var(--token-app-footer-text-color);
+  border-top: 1px solid var(--token-app-footer-border-color);
+}
+
+// topContent structure is temporary, will be refactored when final design is ready
+.hds-app-footer__top-content {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  justify-content: flex-end;
+  width: 100%;
+
+  &:empty {
+    display: none;
```

Comment:
this could be tricky, let's discuss

TLDR: if the consumers add an `{{/if}}` condition for example, the "space" characters used will make the node not empty (it has space characters of comments in it)

Reply:
I explicitly included a comment saying this is temporary. I don't intend to use this code in the final component. I was just experimenting with the visual layout and wanted to minimize refactoring other code until I know what the final design is.

Reply:
I've since deleted this since it was just temporary until I implemented the latest design updates.